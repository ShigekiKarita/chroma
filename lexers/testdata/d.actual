module foo;

class C {}

/* comment1 */
/+ comment2 +/
int main() {
    return 0;
}
