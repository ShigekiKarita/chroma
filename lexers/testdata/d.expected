[
  {"type":"KeywordNamespace","value":"module"},
  {"type":"Text","value":" "},
  {"type":"NameNamespace","value":"foo"},
  {"type":"Operator","value":";"},
  {"type":"Text","value":"\n\n"},
  {"type":"KeywordDeclaration","value":"class"},
  {"type":"Text","value":" "},
  {"type":"NameClass","value":"C"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"{}"},
  {"type":"Text","value":"\n\n"},
  {"type":"CommentMultiline","value":"/* comment1 */"},
  {"type":"Text","value":"\n"},
  {"type":"CommentMultiline","value":"/+ comment2 +/"},
  {"type":"Text","value":"\n"},
  {"type":"KeywordType","value":"int"},
  {"type":"Text","value":" "},
  {"type":"NameFunction","value":"main"},
  {"type":"Operator","value":"()"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"{"},
  {"type":"Text","value":"\n    "},
  {"type":"Keyword","value":"return"},
  {"type":"Text","value":" "},
  {"type":"LiteralNumberInteger","value":"0"},
  {"type":"Operator","value":";"},
  {"type":"Text","value":"\n"},
  {"type":"Operator","value":"}"},
  {"type":"Text","value":"\n"}
]
